ios/ReactNativePageView.m

```diff
@@ -222,6 +222,7 @@ - (instancetype)initWithEventDispatcher:(RCTEventDispatcher *)eventDispatcher {
         _dismissKeyboard = UIScrollViewKeyboardDismissModeNone;
         _coalescingKey = 0;
         _eventDispatcher = eventDispatcher;
+        _overdrag = YES;
```

Comment:
does that mean that the `overdrag` is set to true by default?

Reply:
Yes, it’s a default behaviour on iOS